merchant-sdk/sdk/src/doc/source/event-tracking.rst

```diff
@@ -4,63 +4,16 @@ Event Tracking
+   * ``NoAction`` when the payment flow is finished and the user is being redirected to the bank to pay.
+   * ``OnLoading`` when the payment details are being validated.
+   * ``OnScreenDisplayed(displayedScreen)`` when there is a change in the screens displayed within the payment flow.
+   * ``OnFinishedWithPaymentRequestCreated(paymentRequestId, paymentProviderName)`` when the payment request is ready and the user can be redirected to the bank.
+   * ``OnFinishedWithCancellation`` when the payment flow was cancelled. Can be either from an internal error, or from exiting the payment flow without finalising it.
+   * ``OnErrorOccurred(throwable)`` when there was an error within the payment flow
 
-PaymentComponent
-----------------
+.. note::
 
-The ``PaymentComponent`` class also exposes kotlin flows which you can collect to track events. The following flows are available:
-
-* ``paymentProviderAppsFlow`` is a ``StateFlow`` of ``PaymentProviderAppsState`` which emits the available payment provider apps used by
-  the ``PaymentComponentView`` and related screens. It emits the following states:
-   * ``PaymentProviderAppsState.Loading()`` when the payment provider apps are being loaded.
-   * ``PaymentProviderAppsState.Success(paymentProviderApps)`` when the list of payment provider apps is available.
-   * ``PaymentProviderAppsState.Error(throwable)`` when there was an error loading the payment provider apps.
-* ``selectedPaymentProviderAppFlow`` is a ``StateFlow`` of ``SelectedPaymentProviderAppState`` which emits selected payment provider app shown in
-  the ``PaymentComponentView`` and related screens. It emits the following states:
-   * ``SelectedPaymentProviderAppState.NothingSelected()`` when there is no selection.
-   * ``SelectedPaymentProviderAppState.AppSelected(paymentProviderApp)`` when a payment provider app has been selected.
-
-ReviewFragment
---------------
-
-To get informed of ``ReviewFragment`` events (like the user clicking the "close" or "next" button) you can implement
-the ``ReviewFragmentListener`` and set it on the fragment.
-
-.. code-block:: kotlin
-
-    val reviewConfiguration = ReviewConfiguration(...)
-
-    val paymentReviewFragment = paymentComponent.getPaymentReviewFragment(
-        documentId, reviewConfiguration
-    )
-
-    paymentReviewFragment.listener = object : ReviewFragmentListener {
-        override fun onCloseReview() {
-            // Called only when the ``ReviewConfiguration.showCloseButton`` was set to ``true``.
-            // Dismiss the ReviewFragment.
-        }
-
-        override fun onToTheBankButtonClicked(paymentProviderName: String) {
-            // Log or track the used payment provider name.
-            // No action required, the payment process is handled by the Gini Merchant SDK.
-        }
-    }
+    ``OnFinishedWithPaymentRequestCreated`` and ``OnErrorOccurred`` can be checked for to remove the ``PaymentFragment`` from the hierarchy.
```

Comment:
[Clarification] I think the terminal events are the two `OnFinished...` ones, no?
```suggestion
    ``OnFinishedWithPaymentRequestCreated`` and ``OnFinishedWithCancellation`` can be checked for to remove the ``PaymentFragment`` from the hierarchy.
```